{
	"name": "CVE-2019-16729",
	"description": "pam-python before 1.0.7-1 has an issue in regard to the default environment variable handling of Python, which could allow for local root escalation in certain PAM setups.",
	"packages": [
		{
			"name": "pam-python",
			"releases": {
				"bookworm": {
					"status": "resolved",
					"repositories": {
						"bookworm": "1.0.9-2"
					},
					"fixed_version": "1.0.7-1",
					"urgency": "not yet assigned"
				},
				"bullseye": {
					"status": "resolved",
					"repositories": {
						"bullseye": "1.0.9-2"
					},
					"fixed_version": "1.0.7-1",
					"urgency": "not yet assigned"
				},
				"buster": {
					"status": "resolved",
					"repositories": {
						"buster": "1.0.6-1.1+deb10u1",
						"buster-security": "1.0.6-1.1+deb10u1"
					},
					"fixed_version": "1.0.6-1.1+deb10u1",
					"urgency": "not yet assigned"
				},
				"sid": {
					"status": "resolved",
					"repositories": {
						"sid": "1.0.9-2"
					},
					"fixed_version": "1.0.7-1",
					"urgency": "not yet assigned"
				},
				"stretch": {
					"status": "resolved",
					"repositories": {
						"stretch": "1.0.6-1.1+deb9u1",
						"stretch-security": "1.0.6-1.1+deb9u1"
					},
					"fixed_version": "1.0.6-1.1+deb9u1",
					"urgency": "not yet assigned"
				}
			}
		}
	],
	"scope": "local"
}